v1.5/reference/calicoctl/endpoint.md

```diff
@@ -135,13 +135,13 @@ The command allows you to:
  with the endpoint.
 
 If you are controlling Calico network policy for Docker containers using the
-default networking (i.e. not libnetwork) it is more common to use the  
+default networking (i.e. not libnetwork) it is more common to use the
 `calicoctl container <CONTAINER> profile (set|append|remove) <PROFILES>`
 command, which can be found in the
 [`calicoctl container` reference documentation](./container).
 
 For more information about policy profiles, check out the section on
-[security profile data in the etcd data model documentation](http://docs.projectcalico.org/en/latest/etcd-data-model.html#security-profile-data).
+[security profile data in the etcd data model documentation](../etcd/data-model#security-profiles).
```

Comment:
can this be replaced with the full URL? I've been consistently using that approach so that when considering where to move / break up a document, you can easily search for all docs that reference it